src/pages/workspaces/workspace/Analyses.js

```diff
@@ -145,70 +138,47 @@ const AnalysisCard = ({ namespace, name, updated, metadata, toolLabel, listView,
   }, [
     h(Link, { 'aria-label': 'Analyses menu', onClick: e => e.preventDefault() }, [
       icon('ellipsis-v', {
-        size: listView ? 18 : 16
+        size: analysisContextMenuSize
       })
     ])
   ])
 
-  const title = div({
+  //the flex values for columns here correspond to the flex values in the header
+  const artefactName = div({
     title: getDisplayName(name),
-    style: _.merge({
-      ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto'
-    }, listView ? {
-      marginLeft: '1rem', flexGrow: 1
-    } : { height: 60, padding: '1rem' })
-  }, getDisplayName(name))
+    style: {
+      ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto', flex: 5, textAlign: 'left'
+    }
+  }, [getDisplayName(name)])
 
-  const appIconSrc = Utils.switchCase(toolLabel, [tools.Jupyter.label, () => jupyterLogo], [tools.RStudio.label, () => rLogo])
-  const appIcon = div({ style: { marginRight: '1rem' } }, [
-    img({ src: appIconSrc, style: { height: 40, width: 40 } })
+  const toolIconSrc = Utils.switchCase(application, [tools.Jupyter.label, () => jupyterLogo], [tools.RStudio.label, () => rLogo])
```

Comment:
Nit: I personally find it easier to read when conditionals are on their own lines  🤷🏼 

```javascript
const toolIconSrc = Utils.switchCase(application, 
    [tools.Jupyter.label, () => jupyterLogo],
    [tools.RStudio.label, () => rLogo])
```